.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,44 @@
+name: Release
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI
+    runs-on: ubuntu-latest
+    if: ((github.event_name == 'release') || contains(github.event.pull_request.labels.*.name, 'Build wheels'))
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+
+    - name: Install python-build and twine
+      run: python -m pip install build "twine>=3.3"
+
+    - name: Build package
+      run: python -m build --sdist --wheel .
+
+    - name: List result
+      run: ls -l dist
+
+    - name: Check long_description
+      run: python -m twine check --strict dist/*
+
+    - name: Test package
+      run: |
+        cd ..
+        python -m venv testenv
+        testenv/bin/pip install pytest pytest-remotedata pytest-doctestplus/dist/*.whl
+        testenv/bin/pytest pytest-doctestplus/tests --doctest-plus --doctest-rst
+    - name: Publish distribution 📦 to PyPI
+      if: github.event_name == 'release'
+      uses: pypa/gh-action-pypi-publish@release/v1
+      with:
+        user: __token__
+        password: ${{ secrets.pypi_password }}
```

Comment:
I cannot check but make sure you have `pypi_password` set up as repo secret here.